pingcap/tidb-operator
# Add pods disaster tolerance and data regions disaster tolerance test cases

### What problem does this PR solve? 
- Add test case that checks PD/TIKV/TIDB support disaster tolerance
- Add test case that checks data regions support disaster tolerance
- stability test support to test the branch which has not merged to master
### What is changed and how it works?

### Check List 

Tests 

 - E2E test [done]
 - Stability test [done]
 - Manual test [done]

Code changes

 - Has Helm charts change
 - Has Go code change (package: `tests/`)
 - Makefile

Side effects

Related changes
base PR: #475
should be merged after #499 
### Does this PR introduce a user-facing change?:
 
 ```release-note
None
 ```
